components/deployhq/sources/deploy-complete/deploy-complete.mjs

```diff
@@ -0,0 +1,29 @@
+import common, { STATUS_COMPLETE } from "../common.mjs";
+
+export default {
+  ...common,
+  key: "deployhq-deploy-complete",
+  name: "Deploy Complete",
+  version: "0.0.1",
+  description: "Emit new events when deploys complete",
+  type: "source",
+  props: {
+    http: "$.interface.http",
+  },
+  async run(request) {
+    const { payload } = this.validateRequest(request);
+
+    if (payload.status !== STATUS_COMPLETE) {
+      console.log(`Deploy status [${payload.status}] is not ${STATUS_COMPLETE}`);
+      return;
+    }
+
+    this.$emit({
+      payload,
+    }, {
```

Comment:
Can we just emit `payload` directly?

```suggestion
    this.$emit(payload, {
```

Reply:
I decided to emit the payload without the signature, but as an object to allow adding the signature later if needed without it being a breaking change.

I'm happy to change it if you think it's best?

Reply:
I see. No problem doing the way it is right now!